Dockerfile

```diff
@@ -15,14 +19,14 @@ RUN set -xe; \
 	mkdir -p /var/www/docroot; \
 	cp -R /usr/share/nginx/html/. /var/www/docroot
 
-ARG GOMPLATE_VERSION=3.0.0
+ARG GOMPLATE_VERSION=3.10.0
 
 # Install gomplate
 RUN set -xe; \
 	apk add --no-cache -t .fetch-deps \
 		curl \
 	; \
-	curl -sSL https://github.com/hairyhenderson/gomplate/releases/download/v${GOMPLATE_VERSION}/gomplate_linux-amd64-slim -o /usr/local/bin/gomplate; \
+	curl -sSL https://github.com/hairyhenderson/gomplate/releases/download/v${GOMPLATE_VERSION}/gomplate_linux-${TARGETARCH} -o /usr/local/bin/gomplate; \
```

Comment:
The reasons to remove `slim`:
- https://github.com/hairyhenderson/gomplate/issues/861
- https://github.com/hairyhenderson/gomplate/issues/1085